File: 1-9/2/main.js
let a = 0, b = 1;
let sum = 0;

for (let next = a + b; next < 4_000_000; next = a + b) {
    if (next % 2 === 0) sum += next;
    a = b;
    b = next;
}

console.log(sum)

File: 1-9/2/main.pl
my ($a, $b) = (0, 1);
my $sum = 0;

while (1) {
    my $next = $a + $b;
    last if $next >= 4_000_000;

    $sum += $next if $next % 2 == 0;

    ($a, $b) = ($b, $next);
}

print "$sum\n";
